### PR TITLE
adds testing endpoints

### PR DESCRIPTION
**Pull request by James-Hagerman**
# Description

Replace with description of PR

## Type of change

- [✔ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [✔ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [✔ ] No
- [ ] Not necessary

## Checklist

- [✔ ] My code follows the style guidelines of this project
- [✔ ] I have performed a self-review of my own code
- [✔ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [✔ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [✔ ] There are no merge conflicts

### CODE REVIEW

File: app/api/submission.py [hunk 1]
```diff
@@ -1,104 +1,179 @@
...
-    confidence_flags = []
-    # unpack links for files in submission object
-    for page_num in sub.Pages:
-        # re-init the sha algorithm every file that is processed
-        hash = sha512()
+    '''
+    This is a test endpoint that is a duplicate of the illustration submission endpoint 
+    that allows the web dev team to test the endpoints.
+    '''
         # fetch file from s3 bucket
```
Comment (jproctor-rebecca) [inline app/api/submission.py:28]: thank you for adding the inline documentation - it is the small attention to details like this that create transfer of knowledge in the moment and through time on a project

File: app/api/submission.py [hunk 2]
```diff
@@ -1,104 +1,179 @@
...
+#     try:
+#         assert hash.hexdigest() == sub.Checksum
+#     except AssertionError:
+#         # return bad hash error with the status_code to an ill-formed request
+#         return JSONResponse(status_code=422, content={"ERROR": "BAD CHECKSUM"})
+#     # pass file to the GoogleAPI object to safe search filter the file
+#     response = await vision.detect_safe_search(r.content)
+#     # respond with the output dictionary that is returned from
+#     # GoogleAPI.transcribe() method
+#     return JSONResponse(status_code=200, content=response)
```
Comment (jproctor-rebecca) [inline app/api/submission.py:179]: Do we need to leave commented-out code in the code? or should it be cleaned up?

File: app/api/submission.py [hunk 3]
```diff
@@ -1,104 +1,179 @@
...
-            "IsFlagged": flagged,
-            "LowConfidence": True in confidence_flags,
-            "Complexity": score,
-        },
+            "Complexity": prediction
+        }
     )
 
 
+
```
Comment (jproctor-rebecca) [inline app/api/submission.py:53]: lines 54-108 
during product review, we decided to leave this connected to the GV API and just get the credentials until MVP has been met on the text model
please add this code back
thank you

**Review by jproctor-rebecca — CHANGES_REQUESTED** [review]

File: app/api/submission.py [hunk 4]
```diff
@@ -1,104 +1,179 @@
...
+#         },
+#     )
+
+
+
+
 @router.post("/submission/illustration")
 async def submission_illustration(sub: ImageSubmission):
-    """Function that checks the illustration against the Google Vision
-    SafeSearch API and flags if explicit content detected.
```
Comment (EEdwardsA) [inline app/api/submission.py:78]: Are we sure we want to remove SafeSearch functionality?

**Pull request closed without merging**